common/config/env.go

```diff
@@ -16,9 +16,56 @@
 
 package config
 
-import "os"
+import (
+	"github.com/michaelquigley/pfxlog"
+	"os"
+	"runtime"
+	"sync"
+)
+
+var ensureTmpDirEnvOnce sync.Once
+
+func EnsureTempDirEnv() {
+	defaultTemp := "/tmp"
```

Comment:
what's wrong with just `os.TempDir()`?

Reply:
I didn't even know that existed. Updating.